embedx/config.schema.json

```diff
@@ -435,6 +435,12 @@
         },
         "requested_claims": {
           "$ref": "#/definitions/OIDCClaims"
+        },
+        "skip_audience_check": {
```

Comment:
Why do we need this?

Reply:
This was just a workaround to the following problem.
go-oidc simply compares aud to client id. But Apple puts different values to the audience field for iOS and web.
They use AppId for iOS and ServiceId (ClientId) for web.

So, the best solution would be to extend go-oidc to check list of allowed aud values instead of single ClientId value.

For now I'm removing `skip_audience_check` parameter from this PR. But be aware that iOS clients will not work until go-oidc is extended.

Reply:
> go-oidc simply compares aud to client id. But Apple puts different values to the audience field for iOS and web.
They use AppId for iOS and ServiceId (ClientId) for web.

I see, we then definitely need to add the ability to use a custom audience instead. Do you have an example you could show so I understand the problem? Unfortunately I don't have any mobile programming experience with this. 

Is it not possible to register to apple providers, so one for web and one for iOS in Ory Kratos?

Reply:
> Do you have an example you could show so I understand the problem?


Apple recommends these identifiers to be in a reverse domain name notation. But you can also use any unique string.

> Is it not possible to register to apple providers, so one for web and one for iOS in Ory Kratos?


Should be possible. Not sure what consequences it will have but this will make app management more complex.
And this might not be an option for those who migrates existing apps to Kratos.

I have also created an issue to go-oidc project: https://github.com/coreos/go-oidc/issues/334

Reply:
I'm adding `audiences` list into provider config. If the list is empty, then the audience will be checked against client id, as it's implemented now. So, this will not be a breaking change.
@aeneasr please let me know if you see any disadvantages in such solution.